pkg/apis/config/v1alpha4/types.go

```diff
@@ -76,6 +76,9 @@ type Cluster struct {
 	// in the order listed.
 	// These should be YAML or JSON formatting RFC 6902 JSON patches
 	ContainerdConfigPatchesJSON6902 []string `yaml:"containerdConfigPatchesJSON6902,omitempty"`
+
+	// CgroupRoot specifies the cgroup root to configure the kubelet to use
+	CgroupRoot string `yaml:"cgroupRoot,omitempty"`
```

Comment:
Is there a reason for this to be configurable, instead of us just picking one?

Reply:
Yeah, I guess not, we can just pick one. I'll update it!